app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> str:
+    command_split = command.split()
+
+    if len(command_split) < 3:
+        return "Error. One of the required values is missing"
+
+    mv_command, source_file, dir_and_file = command_split
+
+    if mv_command != "mv":
+        return "Input command is incorrect"
+
+    dir_line = dir_and_file.split("/")[:-1]
+    new_file = dir_and_file.split("/")[-1]
```

Comment:
Use os.path.split

Reply:
Splitting by "/" is a bad practice because in Windows url parts are separated by "\\"